app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module DashboardHelper
+  def dashboard_cards_by_role
+    {
+      challenge_manager: [
+        { image_path: 'folder_open', href: 'manage_submissions', alt: 'submissions', title: 'Submissions',
+          subtitle: 'Manage submissions, evaluations, and evaluators.' },
+        { image_path: 'content_copy', href: 'evaluation_forms', alt: 'evaluation forms', title: 'Evaluation Forms',
+          subtitle: 'Create and manage evaluation forms.' },
+        { image_path: 'map', href: 'user_guide', alt: 'user guides', title: 'User Guides',
+          subtitle: 'Learn how to make the most of the platform.' },
+        { image_path: 'support_agent', href: 'federal-agency-faqs', alt: 'help', title: 'Help',
+          subtitle: 'Get support on the Challenge.Gov platform.' }
+      ],
+      solver: [],
+      evaluator: []
```

Comment:
can you add the evaluator menu?
![Screenshot 2024-09-06 at 5 05 54 PM](https://github.com/user-attachments/assets/daf6f421-1b48-434a-a980-45e5bc5b1379)